napalm/ios/ios.py

```diff
@@ -2439,9 +2439,7 @@ def get_mac_address_table(self):
         )  # 7 fields
         RE_MACTABLE_6500_2 = r"^{}\s+{}\s+".format(VLAN_REGEX, MAC_REGEX)  # 6 fields
         RE_MACTABLE_6500_3 = r"^\s{51}\S+"  # Fill down prior
-        RE_MACTABLE_6500_4 = r"^R\s+{}\s+.*Router".format(
```

Comment:
This line was failing with `napalm/ios/ios.py napalm/ios/ios.py:2442:1: '...'.format(...) has unused arguments at position(s): 1 [pyflakes]`